config/v1/types.go

```diff
@@ -355,14 +355,14 @@ type RequiredHSTSPolicy struct {
 type MaxAgePolicy struct {
 	// The largest allowed value (in seconds) of the RequiredHSTSPolicy max-age
 	// This value can be left unspecified, in which case no upper limit is enforced.
-	// kubebuilder:validation:minimum=0:maximum=2147483647
+	// kubebuilder:validation:Minimum=0:Maximum=2147483647
```

Comment:
This needs a `+` to have the desired effect:
```suggestion
	// +kubebuilder:validation:Minimum=0:Maximum=2147483647
```